README.md

```diff
@@ -41,7 +41,7 @@ Features
 - Community-provided translations for several languages via [Weblate](https://hosted.weblate.org/engage/mgba).
 - Many, many smaller things.
 
-#### Game Boy mappers
```

Comment:
Please don't change the indent levels throughout this file.

Reply:
Will revert. Sorry.